src/app/components/user/pin.service.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2015 SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+(function () {
+  'use strict';
+
+  angular.module('janusHangouts')
+    .service('PinService', PinService);
+
+  function PinService() {
```

Comment:
@imobachgs Actually, I chose a seperate service for `Pin` because `UserService` uses `angular-local-storage` module for storing user settings. `Pin` should not be stored, I guess. Although, I could store `Pin` locally in a variable, but then, IMHO It would be a bit out of flow. What you think??
Otherwise it's fine for me to merge both if you say so.